src/server/mappings/index_mappings.js

```diff
@@ -37,6 +39,14 @@ export class IndexMappings {
         );
       }
 
+      if (illegal.length) {
+        const props = formatListAsProse(illegal);
+        const owner = pluginId ? `registered by plugin ${pluginId} ` : '';
+        throw new Error(
+          `Property name${props.length > 1 ? 's' : ''} ${props} ${owner}are not allowed to start with an underscore (_)`
```

Comment:
Is there a reason there's no space between `${owner}` and `are`? Also same thing on line 38.

Reply:
Oh, line 36 has the trailing space. Just feels wrong.

Reply:
Nevermind, this is fine.